### DISCUSSION TITLE
2008/07/02/%E8%AE%A1%E5%88%92%E4%B9%A6%E7%9A%84%E5%86%85%E5%AE%B9

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#227] [Announcements]
# 2008/07/02/%E8%AE%A1%E5%88%92%E4%B9%A6%E7%9A%84%E5%86%85%E5%AE%B9

我打算主要写两部分的内容。一是 LabVIEW 初学者容易遇到问题或出错的方面，比如程序的常见错误、代码风格、界面设计、调用DLL库等。二是LabVIEW的一些高级用法，比如 VI Scripting、LvClass、XControl 等。

https://blog.qizhen.xyz/2008/07/02/%E8%AE%A1%E5%88%92%E4%B9%A6%E7%9A%84%E5%86%85%E5%AE%B9

**Comment by ruanqizhen** [comment]
之前博客下的留言：

善在琳心
2008年07月16日 10:53 编辑
太期待您这本书啦~~~

回复
